docker/airflow/dags/monkeypox_dag.py

```diff
@@ -0,0 +1,181 @@
+                'sus_deaths' : dict(column=csv_specs.countrycol,
+                                                 date_col=csv_specs.deathdatecol, 
+                                                 dropna=True, entrytype='deaths', dropzeros=True)}
+        Note
+        ----
+        Do not use the same keys conf_aggr and sus_aggr, otherwise your files for suspected entries will overwrite those with confirmed ones
+        Returns
+        -------
+        None
+        """
+        raw_data = pd.read_parquet(path, **read_parquet_kwargs)
+        sus = raw_data[raw_data[csv_specs.statuscol] == csv_specs.statusvals['suspected']]
+        if not len(raw_data):
+            raise ValueError('No data to aggregate')
+        if not conf_aggr:
+            conf_aggr['cases'] = dict(column=csv_specs.countrycol, date_col=csv_specs.confdatecol,
+                            dropna=True, entrytype='cases', dropzeros=True)
+            conf_aggr['deaths'] = dict(column=csv_specs.countrycol, date_col=csv_specs.deathdatecol,
+                            dropna=True, entrytype='deaths', dropzeros=True)
+        if not sus_aggr:
+            sus_aggr['sus_cases'] = dict(column=csv_specs.countrycol, date_col=csv_specs.entrydatecol,
+                            dropna=True, entrytype='cases', dropzeros=True)
+            sus_aggr['sus_deaths'] =  dict(column=csv_specs.countrycol,
+                                             date_col=csv_specs.deathdatecol, # NB. there are some deaths with no deathdate. 
+                                             # right now they are being excluded. We could plot them with the date of entry or of last modification
+                                             dropna=True, entrytype='deaths', dropzeros=True)
+        d_aggr = {}
+        for k, v in conf_aggr.items():
+            d_aggr[k] = group_and_aggr(raw_data, **v)
+            logging.info(f'aggregated {k}')
+        for k, v in sus_aggr.items():
+            d_aggr[k] = group_and_aggr(sus, **v)
+            logging.info(f'aggregated {k}')
+        
+        for k, v in d_aggr.items():
+            outpath = os.path.join(folder, f'{k}.parquet')
+            v.to_parquet(outpath)
+            logging.info(f'Written aggregated parquet file {outpath}')
+
+    transform_and_load(extract(rawfile, folder,
+                               load_cases_kwargs=load_cases_kwargs,
+                               to_parquet_kwargs=to_parquet_kwargs),
+                       folder,
+                       read_parquet_kwargs=read_parquet_kwargs,
+                       conf_aggr=conf_aggr,
+                       sus_aggr=sus_aggr)
+
+cols_needed = [csv_specs.countrycol, csv_specs.confdatecol, csv_specs.deathdatecol, csv_specs.entrydatecol, csv_specs.statuscol]
+dag = etl_mpx('https://raw.githubusercontent.com/globaldothealth/monkeypox/main/latest_deprecated.csv',
+              folder='/home/nr/WORK/MPX-dashboard/data',
```

Comment:
use a subdir of `/tmp` here that's what other Data collection scripts do. They actually configure this sort of things [here](https://github.com/thegraphnetwork/epigraphhub_py/blob/main/epigraphhub/data/data_collection/config.py)

Reply:
I really do not think that it should be a temporary folder, we ought to keep a copy of this raw data. I doubt the size of the files is a problem, but if it were, we can store a copy with a lower frequency and/or check if and what was modified. 
I have changed the env.tpl and and the main.yaml to have an environment variable, and the dag to use the environment variable. The name or location can of course be discussed and changed.

Ivan told me he will open a PR to do a minIO.

Reply:
MinIO is coming online this weekend @NicoRicardi. I hope we can finish this last issue and merge this PR next week.